app/assets/javascripts/admin/addon/components/admin-section-landing-item.gjs

```diff
@@ -0,0 +1,83 @@
+import Component from "@glimmer/component";
+import { hash } from "@ember/helper";
+import { LinkTo } from "@ember/routing";
+import dIcon from "discourse-common/helpers/d-icon";
+import i18n from "discourse-common/helpers/i18n";
+import {
+  DangerButton,
+  DefaultButton,
+  PrimaryButton,
+} from "admin/components/admin-page-action-button";
+
+export default class AdminSectionLandingItem extends Component {
+  get title() {
+    if (this.args.titleLabelTranslated) {
+      return this.args.titleLabelTranslated;
+    } else if (this.args.titleLabel) {
+      return i18n(this.args.titleLabel);
+    }
+  }
+
+  get description() {
+    if (this.args.descriptionLabelTranslated) {
+      return this.args.descriptionLabelTranslated;
+    } else if (this.args.descriptionLabel) {
+      return i18n(this.args.descriptionLabel);
+    }
+  }
+
+  get subTitle() {
+    if (this.args.subTitleLabelTranslated) {
+      return this.args.subTitleLabelTranslated;
+    } else if (this.args.subTitleLabel) {
+      return i18n(this.args.subTitleLabel);
+    }
+  }
+
+  get cssClasses() {
+    const classes = ["admin-section-landing-item"];
+    if (this.args.icon) {
+      classes.push("-has-icon");
+    }
+    return classes.join(" ");
+  }
+
+  <template>
+    <div class={{this.cssClasses}} ...attributes>
+      {{#if @imageUrl}}
+        <img class="admin-section-landing-item__image" src={{@imageUrl}} />
+      {{/if}}
+      {{#if @icon}}
+        <div class="admin-section-landing-item__icon">
+          {{dIcon @icon}}
+        </div>
+      {{/if}}
+      <div class="admin-section-landing-item__content">
+        {{#if this.subTitle}}
+          <h4
+            class="admin-section-landing-item__subtitle"
+          >{{this.subTitle}}</h4>
+        {{/if}}
```

Comment:
It feels a bit weird that the subtitle is above the title. What would you say about naming it "keyword" or something similar?
Or is it a mistake and subtitle should be below?

Reply:
It's correct that it should be above, but yeah I agree subtitle is a bit of a weird name for it :thinking: Apparently it's called a tagline, will use that https://www.w3.org/TR/2014/REC-html5-20141028/common-idioms.html#sub-head https://ux.stackexchange.com/questions/103390/name-of-small-text-above-title